src/main/java/net/syscon/elite/repository/jpa/model/OffenderKeyDateAdjustment.java

```diff
@@ -0,0 +1,46 @@
+package net.syscon.elite.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Data
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(of = "id", callSuper = false)
+@Table(name = "OFFENDER_KEY_DATE_ADJUSTS")
+public class OffenderKeyDateAdjustment extends AuditableEntity {
+
+    @Id
+    @Column(name = "OFFENDER_KEY_DATE_ADJUST_ID", nullable = false)
+    private Long id;
+
+    @Column(name = "SENTENCE_ADJUST_CODE", nullable = false)
+    private String sentenceAdjustCode;
+
+    @Column(name = "ADJUST_DAYS")
+    private Integer adjustDays;
+
+    @Column(name = "OFFENDER_BOOK_ID", nullable = false)
+    private Long offenderBookId;
```

Comment:
We have an OffenderBooking object to support this mapping.  You could map/use that ;)